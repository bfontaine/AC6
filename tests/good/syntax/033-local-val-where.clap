val x : int = 

** This is a correct way of mixing "val" and "where"

  val y : int = z in 
    y
  where 
    val z : int = 42 
  end


