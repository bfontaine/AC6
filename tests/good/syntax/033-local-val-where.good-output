Line 1, characters 100-105: Error (Parsing)
  Unknown error.

