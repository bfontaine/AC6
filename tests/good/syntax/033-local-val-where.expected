val x : int =
  val y : int = z in val z : int = 42 in y
